suite/tests/client-interface/drx-scattergather.c

```diff
@@ -154,6 +154,22 @@ test_avx2_vgatherqpd(uint32_t *ref_sparse_test_buf, uint32_t *test_idx32_vec,
 #    define SPARSE_TEST_BUF_SIZE_U32 (SPARSE_FACTOR * ZMM_REG_SIZE / sizeof(uint32_t))
 #    define POISON 0xf
 
+static bool
+test_avx512_mask()
```

Comment:
`(void)` (unlike C++, in C empty parens do *not* imply no args)

Reply:
Fixed!